html/storybook-v4/src/components/nativeButton/nativeButton.js

```diff
@@ -0,0 +1,24 @@
+import { chevronRightIcon } from '../utils/icons';
```

Comment:
use kebab case for file names, `nativeButton` to `native-button.js` across all the files in this PR